templates/commands/render.go

```diff
@@ -48,10 +51,13 @@ const (
 
 type Render struct {
 	cli.BaseCommand
+	automationFlags flags.AutomationFlags
 
 	testFS     renderFS
 	testGetter getterClient
 
+	hasTTY bool
```

Comment:
Should this be a field of runParams instead, along with all the other "parameters" to realRun()?

Reply:
I ilke it, will update